src/game.js

```diff
@@ -1,9 +1,14 @@
 const fs = require('fs');
-class Game{
-    constructor(creator, createdOn, players){
+const { MessageEmbed } = require('discord.js');
+const Player = require('../src/player.js');
+const Role = require('../src/role.js');
+module.exports = class Game{
+    constructor(creator, createdOn, users){
         this.Creator = creator;
         this.CreatedOn = createdOn;
-        this.Players = players;
+        this.Roles = this.loadRoles();
+        this.Players = this.assignPlayerRoles(users);
+        this.messagePlayers();
```

Comment:
Lines 9-11 could be moved to `sheriff.js` if we didn't want the `Game` class doing _too_ much.